_posts/2020-12-22-aws-antipatterns.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: post
+title: AWS Antipatterns
+author: arto
+excerpt: >
+  Cloud platforms offer unprecedented agility, innovation, and speed. Yet I see the same mistakes being repeated over and over again. This blog post is a collection of some things people often get wrong - and some thoughts on how to dodge those mistakes.
+tags:
+ - AWS
+ - Amazon Web Services
+ - Best Practices
+ - Lessons Learned
+ - Antipatterns
+ - Security
+---
+
+![Speed and agility of cloud development can be unsurpassed](/img/aws-antipatterns/aws_rider.png)
+
+Cloud platforms are incredible. For anyone who used to create or operate software running in a traditional on-premises server room, they offer incredible speed in getting ideas to production, and experimentation with new things. However, this speed comes with a cost: Sometimes that cost is money, as AWS bills can sneak up on you. Sometimes the cost may be a bad architecture that gets more expensive as you progress, a badly compromised security that makes you vulnerable for many new attack vectors - or it can be that you are paying for lots of new services just because you can - without getting the benefits you were looking for. 
```

Comment:
How about adding a link to CVE for AWS? 
https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=AWS

Reply:
Mmm, I'd rather not, as security is only one of the topics here, and spooking about that list is kinda out of proportion for this article.